docs/beginner/tutorial3-pipeline/README.md

```diff
@@ -11,7 +11,7 @@ A vertex is a point in 3d space (can also be 2d). These vertices are then bundle
 
 <img src="./tutorial3-pipeline-vertices.png" />
 
-Most modern rendering uses triangles to make simple shapes such as cube, and complex shapes such as person.
+Most modern rendering uses triangles to make all shapes, from simple (such as cubes) to complex (such as people).
```

Comment:
I think I captured the essence of what you were trying to say here, but I'll let you be the judge.